tests/new_react/new_react.exp

```diff
@@ -47,10 +47,6 @@ new_react.js:4:12,33: string
 This type is incompatible with
   new_react.js:18:15,20: number
 
-new_react.js:4:12,33: string
```

Comment:
Yeah, I mentioned this in the commit message for this change. Since we used to unify props, proptypes are checked in both directions. Elsewhere in this expectation the same error exists with the line numbers reversed.